docs/startingguide.rst

```diff
@@ -0,0 +1,379 @@
+    data = data.metpy.parse_cf()
+
+    # Grab lat/lon values from file as unit arrays
+    lats = ds.lat.metpy.unit_array
+    lons = ds.lon.metpy.unit_array
+
+    # Get the valid time
+    vtime = data.Temperature_isobaric.metpy.time[0]
+
+    # Get the 700-hPa heights without manually identifying the vertical coordinate
+    hght_700 = data.Geopotential_height_isobaric.metpy.sel(vertical=700 * units.hPa,
+                                                     time=vtime)
+
+From here, you could make a map of the 700-hPa geopotential heights. We'll discuss how to
+do that in the Plotting section.
+
+++++++++++++
+Calculations
+++++++++++++
+
+Meteorology and atmospheric science are fully-dependent on complex equations and formulas.
+Rather than figuring out how to write them efficiently in Python yourself, MetPy provides
+support for many of the common equations within the field. For the full list, please see the
+`Calculations <api/generated/metpy.calc.html>`_ reference guide. If you don't see the equation
+you're looking for, consider submitting a feature request to MetPy
+`here <https://github.com/Unidata/MetPy/issues/new/choose>`_.
+
+To demonstrate some of the calculations MetPy can do, let's show a simple example:
+
+.. code-block:: python
+
+    import numpy as np
+    from metpy.units import units
+    import metpy.calc as mpcalc
+
+    temperature = [20] * units.degC
+    rel_humidity  = [50] * units.percent
+    print(dewpoint_rh(temperature, rel_humidity))
+
+.. parsed-literal::
+
+    array([9.27008599]) <Unit('degC')>
+
+.. code-block:: python
+
+    speed = np.array([5, 10, 15, 20]) * units.knots
+    direction = np.array([0, 90, 180, 270]) * units.degrees
+    u, v = mpcalc.wind_components(speed, direction)
+    print(u, v)
+
+.. parsed-literal::
+
+    [0 -10 0 20] knot
+    [-5 0 15 0] knot
+
+As discussed above, if you don't provide units to these functions, they will frequently
+fail with the following error:
+
+.. parsed-literal::
+
+    ValueError: `calculation` given arguments with incorrect units: `variable` requires
+    "[`type of unit`]" but given "none". Any variable `x` can be assigned a unit as follows:
+    from metpy.units import units
+    x = x * units.meter / units.second
+
+If you see this error in your code, just attach the appropriate units and you'll be good to go!
+
+++++++++
+Plotting
+++++++++
+
+MetPy contains two special types of meteorological plots, the Skew-T Log-P and Station plots,
+that more general Python plotting packages don't support as readily. Additionally, with the
+goal to replace GEMPAK, MetPy's declarative plotting interface is being actively developed,
+which will make plotting a simple task with straight-forward syntax, similar to GEMPAK.
+
+******
+Skew-T
+******
+
+The Skew-T Log-P diagram is the canonical thermodynamic diagram within meteorology. Using
+:mod:`matplotlib`, MetPy is able to readily create a Skew-T for you:
+
+.. code-block:: python
```

Comment:
If you change this to:
```rst
  .. plot::
     :include-source:
```
the matplotlib sphinx extension should run the code and inline the image for you.

Reply:
Didn't know this existed...nifty